spec/index.html

```diff
@@ -1075,9 +1091,9 @@ <h4>Patterns of RDF entailment (Informative)</h4>
           </tr>
           <tr >
             <td class="othertable"><dfn>rdfD1</dfn></td>
-            <td class="othertable">   xxx aaa <code>"</code>sss<code>"^^</code>ddd <code>.</code> <br/>
+            <td class="othertable">ttt and <code>"</code>sss<code>"^^</code>ddd appears in ttt <br/>
```

Comment:
```suggestion
            <td class="othertable">Any triple ttt such that <code>"</code>sss<code>"^^</code>ddd appears in ttt <br/>
```
I find it clearer to clearly introduce ttt as "a triple", because that's unusual compare to the rest of the table.

Reply:
I think this is a good clarification.

Reply:
strictly speaking, the ttt does not have to be a triple.

Reply:
reading the table, we have "if S contains ttt, then S RDF-entrails ttt[...]" .
How could anything else than a triple be contained in S or RDF-entailed??
What am I missing?

Reply:
Nevermind, I see that you have applied the suggestion :)